src/server/scripts/EasternKingdoms/ZulGurub/boss_hakkar.cpp

```diff
@@ -23,16 +23,19 @@ Category: Zul'Gurub
 */
 
 #include "ScriptMgr.h"
+#include "DBCStructure.h"
```

Comment:
why it's needed?

Reply:
Should not be ObjectAccessor.h?

Reply:
DBCStructure.h is needed for AreaTriggerEntry usage, other scripts don't use it so they don't need the implementation. All the common headers I find are totally unrelated to this script (like SpellInfo.h) so I think including it directly is the best way.

Or I could split the script into three separate scripts and do away with the AreaTriggerEntry argument, but that'd be useless code duplication, wouldn't it?

Reply:
yeah, no problem :)